src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/PushMetadataToBuildAssetRegistry.cs

```diff
@@ -109,6 +112,14 @@ public async Task<bool> PushMetadataAsync(CancellationToken cancellationToken)
             return !Log.HasLoggedErrors;
         }
 
+        private bool IsStableBuild(BuildData finalBuild)
+        {
+            Regex regex = new Regex("^[0-9].[0-9].[0-9]-.*$");
```

Comment:
Maybe just add a comment with a sample of what this Regex matches

Reply:
Ok. I'll add an example.